### PR TITLE
Minor simplifications

### PR DESCRIPTION
**Pull request by kzu**

### CODE REVIEW

**Comment by kzu** [comment]
These are really minor tweaks that make it easier for me to generate a (new) Moq.Legacy nuget that will allow SxS installation with v5 to ease migration.

**Review by stakx — APPROVED** [review]

File: src/Moq/Moq.csproj [hunk 1]
```diff
@@ -35,8 +35,8 @@
 		<PackageTags>moq;tdd;mocking;mocks;unittesting;agile;unittest</PackageTags>
 		<PackageProjectUrl>https://github.com/moq/moq4</PackageProjectUrl>
 		<RepositoryUrl>https://github.com/moq/moq4</RepositoryUrl>
-		<IncludeSymbols>True</IncludeSymbols>
-		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
+		<IncludeSymbols>false</IncludeSymbols>
+		<AllowedOutputExtensionsInPackageBuildOutputFolder>.pdb;$(AllowedOutputExtensionsInPackageBuildOutputFolder)</AllowedOutputExtensionsInPackageBuildOutputFolder>
```
[the remark this answers is not in the document]
Comment (kzu) [reply src/Moq/Moq.csproj]: Honestly, portable PDBs are the future (have been for a while actually), it's OSS and it's supported everywhere (all IDEs/tools that support .NET Core need to support it). I'd even go as far as to say we should probably even use embedded PDBs instead. That should accomodate that weird scenario from way back then :)

**Comment by kzu** [comment]
Switched to embedded symbols to accommodate that scenario where some tooling failed some years ago with a .pdb alongside the .dll :)

**Comment by stakx** [comment]
Well, that would likely have made it completely unusable but it's so long ago, I'm sure VSTS supports portable PDBs by now, no matter where they reside. :-)

**Pull request merged**